### ISSUE TITLE
error code -7 while creating costum dataset from lmdb

### ISSUE THREAD

**Issue by Motherboard** [NVIDIA/DIGITS#1050]
The only thing in the log is 
```
libdc1394 error: Failed to initialize libdc1394
2016-09-08 05:02:13 [INFO ] Total entries: 13303
```

Doesn't show the dimensions.

I was able to load a different lmdb, prepared in a similar way.. tried sudo chmod -R 755, still no luck.

Loaded the lmdb in python to make sure I can read all the images:

```
lmdb_env = lmdb.open('all_bins_transformed_test')
lmdb_txn = lmdb_env.begin()
lmdb_cursor = lmdb_txn.cursor()
datum = caffe.proto.caffe_pb2.Datum()
i = 0
for key, value in lmdb_cursor:
    datum.ParseFromString(value)
    label = datum.label
    data = caffe.io.datum_to_array(datum)
    im = data.astype(np.uint8)
    im = np.transpose(im, (1, 2, 0)) # original (dim, col, row)
    assert(im.shape == (224,224,3))
    i+=1
    print "label ", label
    plt.imshow(im[:,:,::-1])
    plt.show()

print i
```

everything seems in order.. how can I get a better insight regarding the nature of this error?

**Comment by lukeyeager** [comment]
Can you explain what you're doing a little more clearly? Are you trying to upload a prebuilt LMDB to DIGITS?

**Comment by Motherboard** [comment]
Yes, I have built 2 LMDBs, which consists of labels and images.
I have selected to create a new dataset of type "other" and added these LMDBs as the train images and validation images.

The validation LMDB seems to have been processed successfully, showing both the number of images and their dimensions.

But the train LMDB complains about an error with error code -7.. It shows the number of images in the LMDB correctly, but does not show the image width and height.
